apps/src/block_utils.js

```diff
@@ -426,6 +426,20 @@ const interpolateInputs = function (blockly, block, inputs) {
 };
 exports.interpolateInputs = interpolateInputs;
 
+/**
+ * Add pre-labeled inputs
```

Comment:
it's not that hard to infer, but I'd love some jsdoc documentation on the expected structure of the `args` param in particular